vendor/urls/vendor.py

```diff
@@ -33,6 +33,7 @@
     path('customer/subscription/<uuid:uuid>/cancel/', vendor_views.SubscriptionCancelView.as_view(), name="customer-subscription-cancel"),
     path('customer/subscription/update/<uuid:uuid>/payment', vendor_views.SubscriptionUpdatePaymentView.as_view(), name="customer-subscription-update-payment"),
     path('customer/shipping/<int:pk>/update', vendor_views.ShippingAddressUpdateView.as_view(), name="customer-shipping-update"),               # TODO: [GK-3030] Do not use PKs in URLs
+    path('customer/billing/<int:pk>/update', vendor_views.AddressUpdateView.as_view(), name="customer-billing-update"),
```

Comment:
can we use UUIDs instead of PKs?

Reply:
Added uuid for address and CustomerProfile and corresponding migrations